base/process.jl

```diff
@@ -2,22 +2,76 @@
 
 abstract AbstractCmd
 
+# libuv process option flags
+const UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS = UInt8(1 << 2)
+const UV_PROCESS_DETACHED = UInt8(1 << 3)
+const UV_PROCESS_WINDOWS_HIDE = UInt8(1 << 4)
+
 immutable Cmd <: AbstractCmd
     exec::Vector{ByteString}
     ignorestatus::Bool
-    detach::Bool
+    flags::UInt8 # libuv process flags
```

Comment:
It is potentially subtly breaking to change sizes of internal fields. People generally shouldn't be messing with internal fields, but I still lean towards being a bit more future proof than worrying about conserving 3 bytes.